pkgs/data/themes/catppuccin-gtk/default.nix

```diff
@@ -40,6 +40,10 @@ stdenvNoCC.mkDerivation rec {
 
   nativeBuildInputs = [ gtk3 sassc ];
 
+  patches = [
+    ./create-theme.patch
```

Comment:
Give a more descriptive name for this patch, not the mere filename being modified.